packages/nextjs/app/debug/_components/contract/ReadOnlyFunctionForm.tsx

```diff
@@ -77,11 +77,11 @@ export const ReadOnlyFunctionForm = ({
       </p>
       {inputElements}
       <div className="flex justify-between gap-2 flex-wrap">
-        <div className="flex-grow w-4/5">
+        <div className="flex-grow max-w-[80%]">
```

Comment:
Checking in mobile I noticed a couple of nitpicks on Read results:

- The copy button in `<address />` component it's very small for some resolutions.
- The wei/ether format change button jumps to 2nd line.

| | |
|-|-|
| ![receipt-mobile-1](https://github.com/scaffold-eth/scaffold-eth-2/assets/55535804/06ccfaa5-29a8-4ba1-a336-54d885488675) | ![receipt-mobile-2](https://github.com/scaffold-eth/scaffold-eth-2/assets/55535804/8ad19b75-e1c7-4d49-9138-ce585140202e) |

Tried with `<div className="flex-grow max-w-[90%] sm:max-w-[80%]">` but it's not perfect either.

Reply:
Create an issue for Address component https://github.com/scaffold-eth/scaffold-eth-2/issues/863